Apps/Common/SIMNodalConstraint.h

```diff
@@ -77,18 +70,12 @@ void NodalConstraintProcessor<SIM3D>::apply(SIM3D& sim, TopologySet& myEntitys,
 template<class Dim>
 class SIMNodalConstraint : public Dim {
 public:
-  //! \brief Default constructor.
-  //! \param[in] n1 Dimension of the primary solution field
-  //! \param[in] check If \e true, ensure the model is in a right-hand system
-  SIMNodalConstraint(const SIMbase::CharVec& unf,
-                     bool checkRHS=false) :
-    Dim(unf,checkRHS) {}
-
-  //! \brief Empty destructor
+  //! \brief The constructor forwards to the parent class constructor.
+  SIMNodalConstraint(const std::vector<unsigned char>& unf) : Dim(unf) {}
```

Comment:
It meant so, as I thought this was an obsolete option (not many apps use it, maybe only Elasticity and Poisson). It went out "in the dragsug" with this commit as I at an earlier stage had problem with the CharVec usage in this file. But now it seems no longer the case, likely due to other changes later, so I will revert this file change entirely (not related to the scope of this PR).